### PR TITLE
User Interface

### LINKED ISSUE #28: Team1 [1.5pt] List all books, lables etc
- The following options should be available:
  - List all books
  - List all labels (e.g. 'Gift', 'New')
  - Add a book

### PR DESCRIPTION
**Pull request by Chimwemwe-127001**
- Implement user interface for books and labels with reference to **issue** ***#28***

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -35,15 +62,38 @@ def list_all_games
 
   def list_all_genres; end
 
-  def list_all_labels; end
+  def list_all_labels
```
Comment (youmari) [inline app.rb:65]: @Chimwemwe-127001  Nice implementing List labels but I think you forgot to implement add_label so we can add a label

**Review by youmari — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @Chimwemwe-127001 ,

Good job so far!
✔️ Good documentation
✔️ Nice structure
✔️ Clean code
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

File: app.rb (hunk 1 above)
Comment (Chimwemwe-127001) [reply app.rb:66]: Thanks for the insightful review.
I've implemented changes as requested

**Review by youmari — APPROVED** [review]
# status  🟢  approved
Hi, @Chimwemwe-127001 your PR  is ready to be merged again good job 💯

**Pull request merged**